CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this library adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/fluentassert/verify/compare/v1.1.0...HEAD)
 
+- Add `AsError` function to `FailureMessage`type
```

Comment:
Please update the changelog entry

Reply:
Done in latest commit. Sorry for the oversight.